java/src/jmri/util/HelpUtil.java

```diff
@@ -114,19 +116,17 @@ public static void displayHelpRef(String ref) {
         if (preferences.getOpenHelpOnline()) {
             url = "https://www.jmri.org/" + file;
         } else if (preferences.getOpenHelpOnJMRIWebServer()) {
-            WebServerPreferences webServerPreferences =
-                    InstanceManager.getDefault(WebServerPreferences.class);
+            WebServerPreferences webServerPreferences = InstanceManager.getDefault(WebServerPreferences.class);
             String port = Integer.toString(webServerPreferences.getPort());
-            url = "http://localhost:"+port+"/" + file;
-        } else {    // Assume open help on file if no other option is selected
-            String fileName = FileUtil.getProgramPath().replace("\\", "/") + "help/" + localeStr + "/local/stub/" + ref + ".html";
+            url = "http://localhost:" + port + "/" + file;
+        } else { // Assume open help on file if no other option is selected
+            String fileName = FileUtil.getProgramPath().replace("\\",
+                    "/") + "help/" + localeStr + "/local/stub/" + ref + ".html";
```

Comment:
Is this change done by you or by the editor?

For me, 
```
String fileName = FileUtil.getProgramPath().replace("\\", "/")
         + "help/" + localeStr + "/local/stub/" + ref + ".html";
```
is easier to read than
```
String fileName = FileUtil.getProgramPath().replace("\\",
        "/") + "help/" + localeStr + "/local/stub/" + ref + ".html";
```

Reply:
Editor.  I wouldn't waste my time making that change.  And I agree with you, better before.

Reply:
It would be very good if you could turn this off. I have seen a lot of changes of this kind in your PRs and often it seems that one editor changes from A to B, and then in the next PR, another editor? changes from B to A.

These changes makes the review process more difficult since it's more code to be reviewed.

Reply:
Understood